linera-sdk/src/views/aliases.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Aliases for views using the [`ViewStorageContext`].
+
+use super::ViewStorageContext;
+
+/// An alias to [`linera_views::collection_view::ByteCollectionView`] that uses the
+/// WebAssembly specific [`ViewStorageContext`].
```

Comment:
(Nit: I'd write "WebAssembly-specific" with a hyphen everywhere, but I'm not sure.)